### PR TITLE
Added Service, Controller and Route for Vernal

### PR DESCRIPTION
**Pull request by ajaypatel24**

### CODE REVIEW

File: Pipeline/controllers/VernalController.py [hunk 1]
```diff
@@ -2,8 +2,10 @@
 # basically whatever data manipulation, or combination of service calls we need to do will happen here
 # the routing functions will basically call the controller functions to do what needs to be done
 from services.Vernal import *
+
 class Vernal:
 
     @staticmethod
-    def greeting():
-        return greeting()
+    def vernalSimilarityFunction(representative_graphs, dataset):
```
Comment (rayanosseiran) [inline Pipeline/controllers/VernalController.py:9]: name as vernal_similarity_function to be consistent

**Review by rayanosseiran — APPROVED** [review]

File: Pipeline/controllers/VernalController.py (hunk 1 above)
Comment (ajaypatel24) [reply Pipeline/controllers/VernalController.py:9]: sounds good

File: Pipeline/services/Vernal.py [hunk 2]
```diff
@@ -2,9 +2,14 @@
...
-    r = requests.get(url = URL) 
-    data = r.json() 
-    return data
+def vernalSimilarityFunction(representative_graphs, dataset):
+
+    data = {"graphs": str(representative_graphs), "dataset": dataset}
+    headers = {}
+    files = []
+    response = requests.request(
+        "POST", URL + "CompareSequence/", headers=headers, data=data, files=files)
```
Comment (larakollokian) [inline Pipeline/services/Vernal.py]: I told you in your [vernal review](https://github.com/TeamW-P/VeRNAl/pull/6#discussion_r589170078) to change url path name 😤😤 lol

but friendly reminder that it should be `/compare-sequence` so like change to lowercase, add hyphen and lose the trailing slash

 you should change it here and in VeRNAl

File: Pipeline/routes/route.py [hunk 3]
```diff
@@ -113,7 +114,18 @@ def get_graphs_per_module():
 def greeting_rna_migos():
     return RnaMigos.greeting()
 
+@routes.route('/vernal', methods=['POST'])
+def vernal_file():
+    output = ""
```
Comment (larakollokian) [inline Pipeline/routes/route.py:119]: not sure this is necessary in python

**Review by larakollokian — CHANGES_REQUESTED** [review]
lgtm, small changes requested

File: Pipeline/services/Vernal.py (hunk 2 above)
Comment (ajaypatel24) [reply Pipeline/services/Vernal.py]: changes made

**Pull request merged**